### PR TITLE
Change feature name and summary

### PR DESCRIPTION
**Pull request by codeimpl**
This patch change the name and summary of the function to make it
easier for users to understand the features.

Refs: #809, #832

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <string name="etc">Etc</string>
     <string name="external_download_manager">Use external download manager</string>
     <string name="media_features">Media Features</string>
-    <string name="media_features_summary">Use Banana Browser\'s specialized media features</string>
+    <string name="media_features_summary">Use sepcialized media features of Banana Browser</string>
```
Comment (jdragonbae) [inline res/values/strings.xml:34]: Fix to `specialized`

File: res/values/strings.xml [hunk 2]
```diff
@@ -60,8 +60,8 @@
     <string name="secure_login">Secure Login</string>
     <string name="secure_login_summary">Use additional authentications to autofill stored credentials data securely when you try to sign-in on web sites (e.g. Biometrics, Pattern, PIN and so on)</string>
     <string name="toolbar_editor">Toolbar Editor</string>
-    <string name="toolbar_editor_summary">Frequently used buttons can be assigned to the toolbar</string>
-    <string name="update_adblock_filter">Update adblock filter</string>
+    <string name="toolbar_editor_summary">Personalise your browing experience rearranging frequently used buttons on the bottom toolbar</string>
```
Comment (jdragonbae) [inline res/values/strings.xml:63]: Fix `Personalise` to `Personalize`

File: res/values-ko/strings.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <string name="adblock_with_version">광고 차단 (버전: %s)</string>
```
Comment (romandev) [inline res/values-ko/strings.xml:9]: We should fix this as well

File: res/values/strings.xml [hunk 4]
```diff
@@ -9,7 +9,7 @@
     <string name="adblock_with_version">Ad Block (Ver: %s)</string>
     <string name="adblock_summary">Block ads on sites that show intrusive or misleading ads</string>
     <string name="advanced_features">Advanced Features</string>
-    <string name="advanced_features_summary">Update adblock filter, Translate, External download manager...</string>
+    <string name="advanced_features_summary"> AdBlock, Translate, Data saver mode, External download manager...</string>
```
Comment (romandev) [inline res/values/strings.xml:12]: Data reduction mode

File: res/values/strings.xml (hunk 2 above)
Comment (romandev) [reply res/values/strings.xml:63]: The `Pesonalise` is British style but I'm okay with `Personalize`

File: res/values-ko/strings.xml (hunk 3 above)
Comment (romandev) [reply res/values-ko/strings.xml:9]:
```
res/values/strings.xml:8:    <string name="adblock">Ad Block</string>
res/values/strings.xml:9:    <string name="adblock_with_version">Ad Block (Ver: %s)</string>
res/values-ko/strings.xml:8:    <string name="adblock">광고 차단</string>
res/values-ko/strings.xml:9:    <string name="adblock_with_version">광고 차단 (버전: %s)</string>
res/values-ko/strings.xml:10:    <string name="adblock_summary">방해가 되거나 사용자를 현혹하는 광고를 차단합니다</string>
res/values-ko/strings.xml:12:    <string name="advanced_features_summary">광고차단 필터 업데이트, 번역, 외부 다운로드 매니저...</string>
res/values-ko/strings.xml:64:    <string name="update_adblock_filter">광고 차단 필터 업데이트</string>
res/values-ko/strings.xml:65:    <string name="update_adblock_filter_summary">최신 필터로 더욱 강력한 광고차단 기능을 경험하세요</string>
```

File: res/values/strings.xml (hunk 1 above)
Comment (codeimpl) [reply res/values/strings.xml:34]: done! :)

File: res/values/strings.xml (hunk 2 above)
Comment (codeimpl) [reply res/values/strings.xml:63]: done.

File: res/values/strings.xml (hunk 4 above)
Comment (codeimpl) [reply res/values/strings.xml:12]: done.

File: res/values-ko/strings.xml (hunk 3 above)
Comment (codeimpl) [reply res/values-ko/strings.xml:9]: done.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**